.travis.yml

```diff
@@ -1,24 +1,33 @@
 language: python
-matrix:
-  include:
-    - python: 2.7
-      env: SYMPY_VERSION="oldest"
-    - python: 2.7
-      env: SYMPY_VERSION="latest"
-    - python: 2.7
-      env: SYMPY_VERSION="master"
-virtualenv:
-  system_site_packages: true
+env:
+  matrix:
+  - SYMPY_VERSION="latest"
+  - SYMPY_VERSION="master"
+  - SYMPY_VERSION="oldest"
+python:
+  - 2.7
+  - "2.7_with_system_site_packages"
 before_install:
-  - sudo apt-get update
+  - sudo apt-get update -qq
   # Remove the version of NumPy pre-installed into the Travis virtualenv so
   # that we test with the system installed version.
-  - pip uninstall -y numpy
+  - pip uninstall -y numpy;
+  - if [[ $TRAVIS_PYTHON_VERSION != "2.7_with_system_site_packages" ]]; then
+      sudo apt-get install -qq libatlas-dev libatlas-base-dev liblapack-dev gfortran;
```

Comment:
There are not required for scipy install from apt or conda because those are binary installations. (This is a good reason to switch to conda, because compiling numpy and scipy get old real quick, especially if you use environments or need different versions.)